src/main/java/com/exadel/movie_new/service/MovieServiceImpl.java

```diff
@@ -28,31 +39,43 @@ public Movie addMovie(Movie movie) {
     @Override
     public List<Movie> getAllMovies() {
         List<Movie> movieData = readJsonData();
+        if (CollectionUtils.isEmpty(movieData)) {

```

Comment:
In case if there are no movies yes - it's not an exception, it's just movie has not been added yet ;) You need to return an empty list instead of error.